pom.xml

```diff
@@ -94,6 +94,7 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-mongodb</artifactId>
+			<version>2.2.5.RELEASE</version>
```

Comment:
Nina mentioned something about changing pom.xml. may want to talk to her about this